Source/WebCore/html/shadow/ShadowPseudoIds.h

```diff
@@ -43,6 +43,12 @@ const AtomString& webkitCreditCardAutoFillButton();
 const AtomString& webkitLoadingAutoFillButton();
 const AtomString& webkitStrongPasswordAutoFillButton();
 
+const AtomString& webkitAttachmentContainer();
+const AtomString& webkitAttachmentInfo();
+const AtomString& webkitAttachmentPreview();
+const AtomString& webkitAttachmentSubtitle();
+const AtomString& webkitAttachmentTitle();
```

Comment:
can we guard them behind `ENABLE(ATTACHMENT_ELEMENT)`? We should absolutely not expose these to the web.

Reply:
I'm moving away from pseudo-ids, so these will be removed.